src/main.rs

```diff
@@ -492,6 +492,14 @@ fn main() -> Result<()> {
                     actor_opt = Some(idx as u8)
                 }
             }
+            // can not find actor index by name, return available players name
+            if actor_opt.is_none() {
```

Comment:
You can simply use `anyhow::ensure!` here.

```rust
ensure!(
    actor_opt.is_some(),
    "there are no player with name \"{actor_name}\", available players: {}",
    raw_log.get_names().join(", "),
);
```

Reply:
done